notary-eth-integration-test/src/integration-test/kotlin/integration/eth/DepositMultiIntegrationTest.kt

```diff
@@ -95,8 +97,10 @@ class DepositMultiIntegrationTest {
             val initialAmount = integrationHelper.getIrohaAccountBalance(clientIrohaAccountId, etherAssetId)
             val amount = BigInteger.valueOf(1_234_000_000_000)
             // send ETH
-            integrationHelper.sendEth(amount, relayWallet)
-            integrationHelper.waitOneIrohaBlock()
+            runBlocking { delay(2000) }
```

Comment:
Why is there a delay but it is not present in other tests?

Reply:
Because only this test started to fail due to the fact that it was faster that iroha